web/src/components/countrytable.js

```diff
@@ -455,6 +443,29 @@ const CountryTable = ({
   const [productionTooltip, setProductionTooltip] = useState(null);
   const [exchangeTooltip, setExchangeTooltip] = useState(null);
 
+  // Use the whole history to determine the min/max,
+  // fallback on current data
+  const history = useCurrentZoneHistory() || [data];
+  const [minPower, maxPower] = useMemo(
```

Comment:
Is there a reason why this part is not being calculated in here instead of deeper in `CountryElectricityProductionTable` (which is the only place where it's being used)?

If we moved it there, we wouldn't need to pass any extra props and it wouldn't be calculated also when `CountryCarbonEmissionsTable` is rendered instead.

Reply:
you're absolutely right! Thanks for flagging this!